src/Drupal/DrupalExtension/Context/DrupalContext.php

```diff
@@ -631,9 +631,16 @@ private function getScreenshotParameter(string $name): ?string {
    *
    * @Given I am logged in as user :name
    */
-  public function iAmLoggedInAsUser($name): void {
-    $user = user_load_by_name($name);
-    $this->getSession()->visit(user_pass_reset_url($user) . '/login');
-  }
+  public function iAmLoggedInAsUser($name) {
+
+    //Another solution using user_pass_reset_url() is independent from drush,
+    // but it works only once.
+    $uli = $this->getDriver('drush')->drush('user:login', [
```

Comment:
@kumikoono What `$uli` stands for?  User log-in?  Or typo of `uri`?  Can you change the variable name which makes even more clear meaning?

Reply:
It meant "uli" as it aligns with the alias of "`drush user:login`",  `drush uli.`  
https://drushcommands.com/drush-9x/user/user:login/

Anyway, I renamed it "user_login"